samples/qa-virtuallistnative/src/views/MainPanel.module.less

```
@@ -0,0 +1,11 @@
+.header {
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+}
+
+.list {
+	display: flex;
+	width: 100%;
+	height: 100%;
```

Could you please change height of the list too? 
```suggestion
	height: 90%;
```

---

Suggested change.